### PR TITLE
Add story component

### PR DESCRIPTION
**Pull request by Azmaeus**
Page two of my about-me site is a short history of me.

BEFORE:
N/A

AFTER:
image

### CODE REVIEW

**Comment by Azmaeus** [comment]
The test failed because it was expecting one occurrence of "Software Engineer" on the entire page. I'm converting this PR to draft while I look into JS unit testing to learn the right way to be creating tests then I'll fix this and convert it back from draft.

File: src/components/Story.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.App-story p {
+  margin: 1rem auto;
+  font-size: 1.5rem;
+}
```
Comment (jesseolsen) [inline src/components/Story.css]: nit--Git likes to have an empty line at the end of each file otherwise it shows a red circle

**Review by jesseolsen — APPROVED** [review]

File: src/components/Story.css (hunk 1 above)
Comment (jshaver-emsi) [reply src/components/Story.css]: I believe there is a vs code setting for that.  I don't use vscode, so I'm not exactly sure but seems like its: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Comment (Azmaeus) [reply src/components/Story.css]: Whoa, am I seeing double? XD
Thanks for the tip! I found that setting and updated it.

**Pull request merged**